.idea/compiler.xml

```diff
@@ -7,6 +7,7 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
         <module name="git-hw-dao-three" />
+        <module name="mate-academy-github-hw" />
```

Comment:
The `.idea` folder should be not added to GitHub!

Reply:
Please remove it from GitHub and add it to .gitignore.

Reply:
I added all possible ignore  in my .gitignore : 
.idea
.idea/
.idea/*
*.iml
target/*
/.idea/